lib/TokenManager.ts

```diff
@@ -434,6 +435,10 @@ export class TokenManager {
       options._storageEventDelay = options._storageEventDelay || 1000;
     }
 
+    if (!isLocalhost()) {
+      options.expireEarlySeconds = DEFAULT_OPTIONS.expireEarlySeconds;
```

Comment:
We should mention this in the README as well, otherwise they'll get no feedback on why their setting is being ignored